features/features.go

```diff
@@ -64,12 +64,9 @@ const (
 	// GateAutoRefreshHook enables refresh control from snaps via gate-auto-refresh hook.
 	GateAutoRefreshHook
 
-	// QuotaGroups enable creating resource quota groups for snaps via the rest API and cli.
+	// QuotaGroups enables experimental features related to the Quota Groups API.
```

Comment:
I understand the appeal of reusing this given that we need to be backward compatible with the current usage of it. But we probably need to be very clear in our formulation and docs around this, otherwise is potentially confusing.

```
// QuotaGroups enables any current experimental features related to the Quota Groups API, on top of the features
// already graduated past experimental:
//  * journal quotas are still experimental
//  while guota groups creation and management and memory, cpu, quotas are no longer experimental.
```

Reply:
This makes a lot of sense, thanks for this! I'll reformulate the text based on your suggestion